### PR TITLE
Honeycomb structure revamp

### LINKED ISSUE #3531: [Integrations] Poor performance with large set of integrations
**Current Behavior**
The packaged to create the honeycomb style collection of cells on this page (https://layer5.io/cloud-native-management/meshery/integrations/) performs poorly, making the page unusable.

**Desired Situation**
Displace the https://www.npmjs.com/package/honeycomb-grid package  that is currently being used with a new package (or write a new hexagon style grid).

Possibilities:
1. https://www.npmjs.com/package/honeycomb-grid
2. ...

Nice-to-have:
1. Add a loading spinner (like https://www.npmjs.com/package/react-honeycomb-loading-spinner)

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by GaganpreetKaurKalsi**
**Description**

This PR fixes #3531 

**Notes for Reviewers**
Replace [https://www.npmjs.com/package/react-honeycomb](https://www.npmjs.com/package/react-honeycomb) with a custom honeycomb structure code

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 45bdb1209280ec2970b0165d0ff897252977aa0f at: https://63a6ce422f2679537177ce49--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit de593a8e29c17f2082222f75bac61b5ba1a9205b at: https://63a767656dcdaf41a2dc5c8d--layer5.netlify.app

**Comment by leecalcote** [comment]
I’ve taken a recording comparing the performance of your new implementation between Chrome and Safari - https://youtu.be/8SyKF28g7E4

**Comment by Udit-takkar** [comment]
hey @GaganpreetKaurKalsi  can you please also mention what changes you made which was not present in https://www.npmjs.com/package/react-honeycomb and would lead to better performance ?

**Comment by leecalcote** [comment]
> hey @GaganpreetKaurKalsi can you please also mention what changes you made which was not present in https://www.npmjs.com/package/react-honeycomb and would lead to better performance ?

She didn't make any changes to react-honeycomb.

File: src/sections/Meshery/Meshery-integrations/Honeycomb/Honeycomb.style.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+
+export const HoneycombGrid = styled.div`
+  display: flex;
+  text-align: left;
+  ul {
+    font-size: 0;
+    line-height: 0;
+    height: calc(${(props) => props.height}px + 46px);
```
Comment (Chadha93) [inline src/sections/Meshery/Meshery-integrations/Honeycomb/Honeycomb.style.js:9]: This is good. :ok_hand:

**Review by Chadha93 — APPROVED** [review]
LGTM, we need to test this on https://pagespeed.web.dev/ after the merge to confirm the performance improvement improvements.

**Comment by Chadha93** [comment]
Merging. Thank you @GaganpreetKaurKalsi

**Pull request merged**